metaworld/envs/mujoco/sawyer_xyz/sawyer_push_wall_v2.py

```diff
@@ -0,0 +1,204 @@
+            'hand_init_pos': np.array([0, .6, .2]),
+        }
+
+        self.goal = np.array([0.05, 0.8, 0.015])
+
+        self.obj_init_angle = self.init_config['obj_init_angle']
+        self.obj_init_pos = self.init_config['obj_init_pos']
+        self.hand_init_pos = self.init_config['hand_init_pos']
+
+        self.random_init = random_init
+        self.liftThresh = liftThresh
+        self.max_path_length = 150
+
+        self.action_space = Box(
+            np.array([-1, -1, -1, -1]),
+            np.array([1, 1, 1, 1]),
+        )
+
+        self.obj_and_goal_space = Box(
+            np.hstack((obj_low, goal_low)),
+            np.hstack((obj_high, goal_high)),
+        )
+        self.goal_space = Box(np.array(goal_low), np.array(goal_high))
+
+        hand_to_goal_max = self.hand_high - np.array(goal_low)
+        self.observation_space = Box(
+            np.hstack((self.hand_low, obj_low, -hand_to_goal_max)),
+            np.hstack((self.hand_high, obj_high, hand_to_goal_max)),
+        )
+
+        self.num_resets = 0
+        self.reset()
+
+    @property
+    def model_name(self):
+        return get_asset_full_path('sawyer_xyz/sawyer_push_wall_v2.xml')
+
+    def step(self, action):
+        self.set_xyz_action(action[:3])
+        self.do_simulation([action[-1], -action[-1]])
+        # The marker seems to get reset every time you do a simulation
+        self._set_goal_marker(self._state_goal)
+        ob = self._get_obs()
+        obs_dict = self._get_obs_dict()
+        # reward, _, reach_dist, _, push_dist, pickRew, _, placingDist = self.compute_reward(action, obs_dict)
+        reward, reach_dist, push_dist = self.compute_reward(action, obs_dict)
+        success = float(push_dist <= 0.07)
+
+        info = {
+            'reach_dist': reach_dist,
+            'epRew': reward,
+            'goalDist': push_dist,
+            'success': success,
+            'goal': self.goal
+        }
+        self.curr_path_length +=1
+        return ob, reward, False, info
+
+    def _get_obs(self):
+        hand = self.get_endeff_pos()
+        objPos =  self.data.get_geom_xpos('objGeom')
+        hand_to_goal = self._state_goal - hand
+        flat_obs = np.concatenate((hand, objPos, hand_to_goal))
+        return np.concatenate([flat_obs,])
+
+    def _get_obs_dict(self):
+        return dict(
+            state_observation=self._get_obs(),
+            state_desired_goal=self._state_goal,
+            state_achieved_goal=self.data.get_geom_xpos('objGeom'),
+        )
+
+    def _set_goal_marker(self, goal):
+        # self.data.site_xpos[self.model.site_name2id('goal_{}'.format(self.task_type))] = (
+        #     goal[:3]
+        # )
+        # for task_type in self.task_types:
+        #     if task_type != self.task_type:
+        #         self.data.site_xpos[self.model.site_name2id('goal_{}'.format(task_type))] = (
+        #             np.array([10.0, 10.0, 10.0])
+        #         )
```

Comment:
Remove commented code

Reply:
Oh didn't realize it was still there. I removed those lines.